stable/openvpn/templates/openvpn-deployment.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: apps/v1beta2
+apiVersion: apps/v1
```

Comment:
This is the only substantive change here, right?

Reply:
Yes, correct